.github/workflows/templating.yml

```
@@ -0,0 +1,28 @@
+name: Templating
+on:
+  schedule:
+    # Run every Monday at 00:00 UTC
+    - cron: "0 0 * * 1"
+  pull_request:
+    paths:
+      - ".github/workflows/templating.yml"
+  workflow_dispatch: # Allow manual triggering
```

Let's briefly revisit the triggering conditions here.
The underlying templates are only supposed to change when a new version of the action is released.
The respective update PRs will be generated by Renovate automatically.
So I believe the `schedule` component can be removed here.
The `workflows_dispatch` can remain here for manual triggering.
The `pull_request` one is probably the right one to use here as it will trigger then run on any updates to this file.
The only thing we should try to avoid is creating endless loops of updates, if we should ever decide to template this workflow file (which seems likely in my opinion). But maybe the below is fine already.

```suggestion
  pull_request:
    paths:
      - ".github/workflows/templating.yml"
  workflow_dispatch: # Allow manual triggering
```